### PR TITLE
Src/test pdf converter with specifications

### PR DESCRIPTION
**Pull request by JuanCarlosMartinez4**

### CODE REVIEW

File: src/main/java/com/jalasoft/jfc/controller/ImageConverterController.java [hunk 1]
```diff
@@ -57,8 +58,9 @@ public String imageConverter(
             @RequestParam("file") MultipartFile file,  @RequestParam (defaultValue = " ") String md5,
             @RequestParam (defaultValue = CONVERTED_FILE) String outputPathFile, @RequestParam String outputFileName,
             @RequestParam (defaultValue = ".png") String imageFormat, @RequestParam (defaultValue = "false")
-            boolean Thumbnail,  @RequestParam (defaultValue = "0") int ImageWidth, @RequestParam (defaultValue = "0")
-            int ImageHeight, @RequestParam (defaultValue = "0") float degreesToRotate) {
+                    boolean Thumbnail,  @RequestParam (defaultValue = "0") int ImageWidth,
+            @RequestParam (defaultValue = "0") int ImageHeight, @RequestParam (defaultValue = "0") float degreesToRotate)
```
Comment (Amet015) [inline src/main/java/com/jalasoft/jfc/controller/ImageConverterController.java]: please correct this lines you can up  @RequestParam (defaultValue = "0")

File: src/main/java/com/jalasoft/jfc/model/strategy/CommandOutputFileName.java [hunk 2]
```diff
@@ -30,16 +34,25 @@ public CommandOutputFileName(String commandValue) {
...
     /**
-     * Builds command.
-     * @return String of a command.
+     * Build command.
+     * @return string of command.
+     * @throws CommandValueException
      */
-    public String command(){
+    public String command() throws CommandValueException {
+
```
Comment (Amet015) [inline src/main/java/com/jalasoft/jfc/model/strategy/CommandOutputFileName.java]: please delete this line

File: src/main/java/com/jalasoft/jfc/model/pdf/PdfConverter.java [hunk 3]
```diff
@@ -25,10 +25,7 @@
 import com.jalasoft.jfc.model.strategy.CommandImageFormat;
 import com.jalasoft.jfc.model.strategy.ContextStrategy;
 import com.jalasoft.jfc.model.strategy.CommandImageConverter;
-
```
Comment (Amet015) [inline src/main/java/com/jalasoft/jfc/model/pdf/PdfConverter.java:28]: insert a new line to separate libraries

**Review by Amet015 — APPROVED** [review]

**Review by Amet015 — APPROVED** [review]

File: src/main/java/com/jalasoft/jfc/controller/VideoConverterController.java [hunk 4]
```diff
@@ -68,13 +69,12 @@ public String videoConverter(
...
-            @RequestParam (defaultValue = "") short rotate) throws ConvertException, IOException {
+            String frameRate, @RequestParam (defaultValue = "0") int width, @RequestParam (defaultValue = "0")
+            int height, @RequestParam (defaultValue = "") String videoCodec, @RequestParam (defaultValue = "")
+            String audioCodec, @RequestParam (defaultValue = "") String videoBitRate, @RequestParam (defaultValue = "")
+            String audioBitRate, @RequestParam (defaultValue = "-1") int quality, @RequestParam (defaultValue = "0")
+            int channelsNumber, @RequestParam (defaultValue = "") String volume, @RequestParam (defaultValue = "")
+            short rotate) throws CommandValueException {
 
         Md5Checksum md5Checksum = new Md5Checksum();
         Param param = new VideoParam("thirdparty\\FFmpeg\\bin\\ffmpeg.exe");
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/controller/VideoConverterController.java:80]: Here you can use PathJfc class for getting ffmpeg binary

File: src/main/java/com/jalasoft/jfc/model/exception/CommandValueException.java [hunk 5]
```diff
@@ -26,7 +26,7 @@
      * @param message of error.
      */
     public CommandValueException(String message, String param) {
-        this.message = message;
+        this.message = message + " " + param;
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/model/exception/CommandValueException.java:29]: instead of + " " +, you can use SPACE.getCommandValue()

**Pull request merged**